File: SharedModels/Metadata.cs
using System;

namespace SharedModels
{
    public class Metadata
    {
        public string Title { get; set; }

        public string Author { get; set; }

        public string Version { get; set; }

        public DateTime DateGenerated { get; set; }

        public string VersionHash { get; set; }

        public string StartFile { get; set; }
    }
}

File: SM-engine/src/models/Metadata.ts
import { ImageFormatType } from '../enums';

export interface Metadata {
  Title: string;
  TitleScreenImageName: string;
  Author: string;
  Version: string;
  DateGenerated: Date;
  VersionHash: string;
  StartFile: string;
  ScenarioExtension: string;
  PictureFormatType: ImageFormatType;
}
